plume-models/src/blogs.rs

```diff
@@ -431,12 +427,10 @@ impl NewBlog {
             actor_id,
             title,
             summary,
-            outbox_url: String::from(""),
-            inbox_url: String::from(""),
             instance_id,
-            ap_url: String::from(""),
             public_key: String::from_utf8(pub_key).or(Err(Error::Signature))?,
             private_key: Some(String::from_utf8(priv_key).or(Err(Error::Signature))?),
+            ..NewBlog::default()
```

Comment:
I've never used this syntax, but it looks super nice

Reply:
what does it do?

Reply:
For all the fields that were not provided, it takes their value from the given struct (here `NewBlog::default()`).